templates/ubuntu/tpl_megamnilavu_install.go

```diff
@@ -45,13 +45,12 @@ func (m *UbuntuMegamNilavuInstallTemplate) Render(pkg urknall.Package) {
 	pkg.AddCommands("packages",
 		InstallPackages("software-properties-common", "python-software-properties", "ruby2.0", "ruby2.0-dev"),
```

Comment:
remove `ruby2.0` and `ruby2.0-dev` dependency. This will be a manual install for the user.